tasks/move-granules/tests/test-index.js

```diff
@@ -220,9 +223,10 @@ test.serial('Should update filenames with updated S3 URLs.', async (t) => {
 
 test.serial('Should not overwrite CMR fileType if already explicitly set', async (t) => {
   const newPayload = buildPayload(t);
-  addCmrFileToPayload(t, newPayload);
+  //addCmrFileToPayload(t, newPayload);
+  newPayload.input.granules[0].files[3].fileType = 'userSetType';
```

Comment:
Is this something that needs cleanup? If not, did you decide that manually targeting a file by index is safe given that we control the input to these tests?

Reply:
👍 c16d57915